lib/hexpm/accounts/audit_log.ex

```diff
@@ -138,6 +142,16 @@ defmodule Hexpm.Accounts.AuditLog do
   defp extract_params("password.reset.finish", nil), do: %{}
   defp extract_params("password.update", nil), do: %{}
 
+  defp extract_params("billing.create", {organization_name, params}),
+    do: %{
+      organization: organization_name,
```

Comment:
Please do `serialize(organization)` like above.

Reply:
Sure, updated in baedfd1136f1ab926a259b7501f7114bd9cd9be7